### PR TITLE
refactor(input): invalid state validation text

### PR DESCRIPTION
**Pull request by justinhough**
**Related Issue:** #1592

## Summary
Adjusted validation text under input to match Figma.

Adjustments made per latest issue comment:
- [x] Spacing between validation icons and text is 8px in all scales
- [x] validation text is 1 font-size smaller than the label of it's parent in all scales
- [x] validation text is ui-text-1 (#151515) and weight is medium
- [x] validation on scale small has 4px top and bottom margin
- [x] validation on scale medium and large have 8px top and bottom margin

### CODE REVIEW

File: src/components/calcite-input-message/calcite-input-message.scss [hunk 1]
```diff
@@ -64,3 +64,30 @@
...
 }
+
+// Validation Text
+:host([status][active]) {
+  @apply text-color-1 
+    font-medium;
+}
+
+:host([status][active][scale="s"]) {
+  @apply text--3;
```
Comment (caripizza) [inline src/components/calcite-input-message/calcite-input-message.scss:75]: I think the font sizes need the `h` suffix based on what I see in Figma

**Review by caripizza — CHANGES_REQUESTED** [review]
Looking good thus far, can you remove the commented out code?

File: src/components/calcite-input-message/calcite-input-message.scss (hunk 1 above)
Comment (julio8a) [reply src/components/calcite-input-message/calcite-input-message.scss:75]: Yes ^

**Comment by julio8a** [comment]
On Figma, font size is 14px, but here the hint text is 16px. @bstifle can you confirm which one is correct?

**Review by bstifle — APPROVED** [review]
Dude, lookin SO good 🥇

**Review by julio8a — APPROVED** [review]
Looks good!

**Review by caripizza — APPROVED** [review]
👍

**Comment by caripizza** [comment]
(Needs latest master merged in please)

**Pull request merged**